### PR TITLE
Persona: Standardized naming of Persona sizes, all numerical now

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [X] Include a change request file using `$ npm run change`

#### Description of changes

- Persona had a mix of relative/descriptive named sizes (e.g. small, regular, large) and numerically named sizes (e.g. size28). This PR renames all size names to be numerical and deprecates the relative names.
- Used the `@deprecated ` decorator to deprecated the relevant enums, but it looks like the typical `this._warnDeprecations()` call in the the Persona constructor doesn't work as I expect it to because I'm not deprecating props. Thus **it'd be really great to get extra eyes on how I'm deprecating** so we don't get breaks, and it's clear to users.
- Mapped old sizes to include both old and new classNames for the root to not break style overrides folks might be using
- Perhaps other mapping needs to be done, too?
- Updated all components using Persona to the new sizes.
- Updated tests.
- Updated PersonaPage's documentation.

I can definitely add screenshots of all components with Persona if that's helpful, but I'm thinking that screener should catch any visual discrepancies these changes caused. And the intent is not to change visuals - polish will be a separate PR.

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by lynamemi** [comment]
I'm almost shocked it passed screener 😄

File: packages/office-ui-fabric-react/src/components/Persona/Persona.scss [hunk 1]
```diff
@@ -263,12 +263,12 @@ $ms-Persona-presenceSizeXl: 28px;
 
 //== Modifier: Tiny Persona
 //
-.root.rootIsTiny {
-  height: $ms-Persona-sizeTiny;
-  min-width: $ms-Persona-sizeTiny;
+.root.rootIs12 {
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Persona/Persona.scss]: Can you rename all of these to rootIsSizeXX, for more descriptive and consistent class names?

File: packages/office-ui-fabric-react/src/components/Persona/PersonaConsts.ts [hunk 2]
```diff
@@ -7,15 +7,24 @@ import * as stylesImport from './Persona.scss';
...
   [PersonaSize.regular]: '',
-  [PersonaSize.large]: 'ms-Persona--lg ' + styles.rootIsLarge,
-  [PersonaSize.extraLarge]: 'ms-Persona--xl ' + styles.rootIsExtraLarge,
+  [PersonaSize.large]: 'ms-Persona--lg ' + styles.rootIsLarge + styles.rootIs72,
+  [PersonaSize.extraLarge]: 'ms-Persona--xl ' + styles.rootIsExtraLarge + styles.rootIs100,
+
   [PersonaSize.size28]: 'ms-Persona--28 ' + styles.rootIs28,
-  [PersonaSize.size16]: 'ms-Persona--16 ' + styles.rootIs16
+  [PersonaSize.size16]: 'ms-Persona--16 ' + styles.rootIs16,
+  [PersonaSize.size12]: 'ms-Persona--12 ' + styles.rootIs12,
```
Comment (mtennoe) [inline packages/office-ui-fabric-react/src/components/Persona/PersonaConsts.ts]: Can you rename to ms-Persona--sizeXX for more descriptive and consistent names?

**Comment by lynamemi** [comment]
Thank you both for the review! I'll get those names changed. And yeah, I've had bytes dinging me on a couple of PRs, I'll ping @dzearing about it...

**Pull request merged**